### ISSUE TITLE
Great Project!

### ISSUE THREAD

**Issue by kellerassel007** [rhlarora84/aws-saml-login-chrome-extension#1]
Hey, I tested it out and I love it!
I have like 50 AWS accounts available in my SAML login page and this tool is exactly what I was looking for.

Some suggestions for improvements:

1. If the role name is too long, it doesn't fit inside the account box:
![image](https://user-images.githubusercontent.com/17928494/234557734-8149f11f-1470-497c-adbf-51380c1a5928.png)
2. Select and store favorite accounts (or maybe last used x accounts)
4. Select colours for accounts (to highlight PROD accounts)
5. Publish official Chrome extension for easier installation of the extension

**Comment by rhlarora84** [comment]
Thanks for the feedback. I will push out the updates.

It is officially available on Chrome store.
https://chrome.google.com/webstore/detail/samlify-for-aws/nhmhangnpobobcnlfpfblbfkjcoppdng

**Comment by rhlarora84** [comment]
I have published a new version in Chrome store that adds the following feature. Chrome has a review process which usually take  few days before the update starts showing up.
- Mark the Account as Favourite by clicking on the Home Icon. Favourite accounts will be shown first
- Add an icon to copy the account id to the clipboard
- If the Role exceeds a certain length, it will be truncated. Tooltip has been added to show the full name

Hope this helps.
